testing/docker/bookinfo.yaml

```diff
@@ -56,7 +53,7 @@ reviews-v2:
     - A8_CONTROLLER_URL=http://controller:8080
     - A8_CONTROLLER_POLL=5s
     - A8_REGISTRY_POLL=5s
-    - A8_HEALTHCHECKS=http://localhost:9080/health
+    - A8_PROXY_ADAPTER=envoy
   external_links:
```

Comment:
Use env vars.. 
```yaml
 - A8_PROXY_ADAPTER="${PROXY_TYPE}"
```

and use a simple for loop in the build_and_run.sh to iterate over nginx and envoy. Something like this
```bash
for proxy in nginx envoy do
export PROXY_TYPE=$proxy
if [ "$A8_TEST_DOCKER" == "true" ]; then
    $SCRIPTDIR/docker/test-docker.sh
fi

if [ "$A8_TEST_K8S" == "true" ]; then
    $SCRIPTDIR/kubernetes/test-kubernetes.sh
fi
done
```